src/layouts/partials/menu.html

```diff
@@ -53,3 +54,58 @@
 		</ul>
 	</nav>
 </section>
+{{ else }}
+<section class="nav-wrap">
+	<nav class="acnav" role="navigation">
+		<ul class="acnav__list acnav__list--level1">
+
+			{{ $currentPage := . }}
+            {{ range .Site.Menus.principal }}
```

Comment:
So this is the only line that changed. No idea what it does.

Reply:
it renders the new menu when you are not vintage. It will be removed after migration is done (just temporarily)